CI: Travis failed on arm64: TestShuffle FAIL

CI failure Link:
https://travis-ci.com/github/cilium/cilium/jobs/359548956#L13718

```
=== RUN   TestShuffle
    TestShuffle: safe_rand_test.go:68: Shuffle() did not modify s2 in 10 retries
--- FAIL: TestShuffle (0.00s)
FAIL
coverage: 22.9% of statements in ./...
FAIL	github.com/cilium/cilium/pkg/rand	0.018s
FAIL
```

--- a/pkg/rand/safe_rand_test.go
+++ b/pkg/rand/safe_rand_test.go
@@ -44,7 +44,7 @@ func TestShuffle(t *testing.T) {
 	copy(s2, s1)
 
 	var same int
-	for retry := 0; retry < 1; retry++ {
+	for retry := 0; retry < 10; retry++ {
 		same = 0
 
 		r0.Shuffle(len(s2), func(i, j int) {

